<launch>

    <!-- TRO -->
    <node pkg="trajectory_optimizer" type="midline" name="midline" output="screen">
        <rosparam file="$(find trajectory_optimizer)/params/gro.yaml" subst_value="True"/>
    </node>

    <!-- <node name="dynamic_reconfigure_load_tro" pkg="dynamic_reconfigure" type="dynparam" 
          args="load /trajectory_optimizer $(find trajectory_optimizer)/params/dynamic.yaml" />  -->

</launch>